### PR TITLE
customized contributing to match team contract

### LINKED ISSUE #25: CONTRIBUTING.md
Customize contributing.md to match our contract.

### PR DESCRIPTION
**Pull request by ella-irene**
Added a few sections to more specifically describe expected merging / branching workflow.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,6 +69,19 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated.
 3. The pull request should work for all currently supported operating systems and versions of Python.
 
+## Merge Guidelines
+* Do not push anything directly to the MAIN brancg without first opening a pull request
```
Comment (salva-u) [inline CONTRIBUTING.md]:
```suggestion
* Do not push anything directly to the MAIN branch without first opening a pull request
```

**Review by jlee2843 — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,6 +69,19 @@ Before you submit a pull request, check that it meets these guidelines:
...
+* Do not merge with main before all core authors have reviewd and approved the request
```
Comment (salva-u) [inline CONTRIBUTING.md]:
```suggestion
* Do not merge with main before all core authors have reviewed and approved the request
```

**Review by salva-u — CHANGES_REQUESTED** [review]
all good, just 2 suggestions

**Review by Rachel0619 — APPROVED** [review]
LGTM!

**Comment by ella-irene** [comment]
> all good, just 2 suggestions

Thanks for the typo fixes!! Appreciate it.

**Review by salva-u — APPROVED** [review]

**Pull request merged**